internal/pkg/manifest/backend_svc.go

```diff
@@ -100,7 +100,15 @@ func (s *BackendService) Port() (port uint16, ok bool) {
 
 // Publish returns the list of topics where notifications can be published.
 func (s *BackendService) Publish() []Topic {
-	return s.BackendServiceConfig.PublishConfig.Topics
+	var pubs []Topic
+	for _, topic := range s.BackendServiceConfig.PublishConfig.Topics {
+		tempTopic := topic
+		if topic.FIFO.IsEnabled() {
+			tempTopic.Name = aws.String(aws.StringValue(topic.Name) + ".fifo")
+		}
+		pubs = append(pubs, tempTopic)
```

Comment:
nit:
```suggestion
		if topic.FIFO.IsEnabled() {
			topic.Name = aws.String(aws.StringValue(topic.Name) + ".fifo")
		}
		pubs = append(pubs, topic)
```
Because `topic` is passed by value here in the `range` over. Changing `topic.Name` won't affect `s.BackendServiceConfig.PublishConfig.Topics[idx].Name`.